### PR TITLE
Added For Fourm Admins!

### PR DESCRIPTION
**Pull request by AlphaGaming**

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java [hunk 1]
```diff
@@ -9,6 +10,7 @@
 public enum TFM_PlayerRank
 {
     DEVELOPER("a " + ChatColor.DARK_PURPLE + "Developer", ChatColor.DARK_PURPLE + "[Dev]"),
+    ADMIN("a " + ChatColor.RED + "Administrator", ChatColor.RED + "[Administrator]"),
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java]: Change this to:
Executive("a " + ChatColor.RED + "Executive", ChatColor.RED + "[Executive]"),

File: src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java [hunk 2]
```diff
@@ -70,6 +72,11 @@ public static TFM_PlayerRank fromSender(CommandSender sender)
             return DEVELOPER;
         }
 
+        if (ADMIN.contains(sender.getName()))
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java]: Change this to:

 if (Executive.contains(sender.getName()))

File: src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java [hunk 3]
```diff
@@ -70,6 +72,11 @@ public static TFM_PlayerRank fromSender(CommandSender sender)
...
+        {
+            return ADMIN;
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java]: Change this to:

return Executive;

File: src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 import me.StevenLawson.TotalFreedomMod.Config.TFM_ConfigEntry;
 import static me.StevenLawson.TotalFreedomMod.TFM_Util.DEVELOPERS;
+import static me.StevenLawson.TotalFreedomMod.TFM_Util.ADMIN;
```
Comment (DUFCLiam) [inline src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java]: Change this to:

import static me.StevenLawson.TotalFreedomMod.TFM_Util.ADMIN;

But be sure to change this:

public static final List<String> ADMIN = Arrays.asList("cowgomoooo12", "OliverDatGuy");

To:

public static final List<String> EXECUTIVE = Arrays.asList("cowgomoooo12", "OliverDatGuy");

**Comment by mysteriace** [comment]
HUGE VOUCH!

**Comment by OnlineRepz** [comment]
No, Executives will show up as Senior in-game, on forum they would be administrator (Executive)

**Comment by Hockeyfan360** [comment]
@AlphaGaming I don't believe that there is an accepted corresponding issue currently open.

**Comment by JeromSar** [comment]
This PR does not meet the [pull request requirements](https://github.com/TotalFreedom/TotalFreedomMod/wiki/Contributing#pull-request-requirements) and thus is closed.

**Pull request closed without merging**